user_docs/en/userGuide.md

```diff
@@ -2243,6 +2243,36 @@ Disabling this option may improve readability.
 
 To toggle show selection from anywhere, please assign a custom gesture using the [Input Gestures dialog](#InputGestures).
 
+##### Formatting display {#BrailleSettingsFormattingDisplay}
```

Comment:
New anchor names should not contain the name of the settings panel in which they are located, so that they do not become obsolete or irrelevant if the option is moved in the future, e.g. if we split the braille settings panel in two separate categories.
```suggestion
##### Formatting display {#BrailleFormattingDisplay}
```